test/ecs/ecs_metadata/resources/emf_prometheus_redis_schema.json

```diff
@@ -0,0 +1,75 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Prometheus-Redis structured log schema",
+  "description": "JSON schema for the CloudWatch agent ECS structured log",
+  "type": "object",
+  "properties": {
+    "ClusterName": {},
+    "CloudWatchMetrics": {
+      "type": "array",
+      "items": {
+        "type": "object",
+        "additionalProperties": true,
+        "properties": {
+          "Namespace": {
+            "type": "string"
+          },
+          "Dimensions": {
+            "type": "array",
+            "items": {
+              "type": "array",
+              "items": {
+                "type": "string"
+              }
+            }
+          },
+          "Metrics": {
+            "type": "array",
+            "items": {
+              "type": "object",
+              "additionalProperties": true,
+              "properties": {
+                "Name": {
+                  "type": "string"
+                },
+                "Unit": {
+                  "type": "string"
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+    "ECS_PROMETHEUS_EXPORTER_PORT": {},
+    "LaunchType": {},
+    "StartedBy": {},
+    "TaskClusterName": {},
+    "TaskDefinitionFamily": {},
+    "TaskGroup": {},
+    "TaskId": {},
+    "TaskRevision": {},
+    "Timestamp": {},
+    "Version": {},
+    "container_name": {},
+    "job": {},
+    "prom_metric_type": {}
+  },
+  "required": [
+    "ClusterName",
+    "ECS_PROMETHEUS_EXPORTER_PORT",
+    "LaunchType",
+    "StartedBy",
+    "TaskClusterName",
+    "TaskDefinitionFamily",
+    "TaskGroup",
+    "TaskId",
+    "TaskRevision",
+    "Timestamp",
+    "Version",
+    "container_name",
+    "job",
+    "prom_metric_type"
```

Comment:
This property wont exist in public CWA so Id imagine this test fails there?

Reply:
The test passes 🤷

Reply:
Also, to clarify, I grabbed the schema directly from what got published to CloudWatch Logs from my forked integration tests.

Reply:
Yeah my bad.. this does exist even in CWA.